Duplicate Type "defaultRole"

<!--
If you need help with discord.js installation or usage, please go to the discord.js Discord server instead:
  https://discord.gg/bRCvFy9
This issue tracker is only for bug reports and enhancement suggestions.
You won't receive any basic help here.
-->

**Please describe the problem you are having in as much detail as possible:**

The following lines cause an issue when using TypeScript with discord.js as the `defaultRole` attribute is defined twice.

https://github.com/discordjs/discord.js/blob/0b1176d9a1078d056835dc4e573fe639d4c5ed79/typings/index.d.ts#L404-L406

Causing the following semantic errors:

`error TS2300: Duplicate identifier 'defaultRole'`
`error TS2717: Subsequent property declarations must have the same type.  Property 'defaultRole' must be of type 'Role | null', but here has type 'Role'.`

Not sure how this wasn't caught by whatever linter and compiler contributors are using.

**Include a reproducible code sample here, if possible:**

Literally just try running compiling a bot with Typescript.

**Further details:**
- discord.js version: master (0b1176d9a1078d056835dc4e573fe639d4c5ed79)
- Node.js version: 10.15.3 (Dubnium LTS)
- Operating system: Windows 10
- Priority this issue should have – please be realistic and elaborate if possible: Not sure how many people use v12, I just started migrating to it.

<!--
If this applies to you, please check the respective checkbox: [ ] becomes [x].
You don't have to modify the text to suit your particular situation – if you want to
elaborate, please do so in the description.
While it's not a requirement to test your issue on the master branch, it would make fixing
the problem a lot easier for us, so please do so if possible.
-->

- [x] I have also tested the issue on latest master, commit hash: 0b1176d9a1078d056835dc4e573fe639d4c5ed79

--- a/typings/index.d.ts
+++ b/typings/index.d.ts
@@ -403,7 +403,6 @@ declare module 'discord.js' {
 		public readonly createdTimestamp: number;
 		public readonly defaultRole: Role | null;
 		public defaultMessageNotifications: DefaultMessageNotifications | number;
-		public readonly defaultRole: Role;
 		public deleted: boolean;
 		public embedEnabled: boolean;
 		public emojis: GuildEmojiStore;

